pkgs/development/libraries/mesa/default.nix

```diff
@@ -41,11 +40,11 @@ let
     optionals (elem "drm" eglPlatforms)
     (if (stdenv.isAarch32 || stdenv.isAarch64)
     then ["nouveau"]
-    else ["i915" "i965" "nouveau" "radeon" "r200"]);
```

Comment:
X.org couldn't start so the dri `i915` driver should be kept by default instead of the gallium `i915`.